specifyweb/specify/migration_utils/tests/test_bulk_create.py

```
@@ -0,0 +1,82 @@
+        self.assertEqual(result, 2)
+
+        # -----------------------
+        # Assert ORM interaction
+        # -----------------------
+        self.assertTrue(mock_manager.filter.called)
+        self.assertTrue(mock_manager.bulk_create.called)
+
+        # -----------------------
+        # Inspect bulk_create payload
+        # -----------------------
+        args, kwargs = mock_manager.bulk_create.call_args
```

Ruff (0.15.17)

[warning] Unpacked variable `kwargs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)